mixed-content/generic/worker.js

```diff
@@ -0,0 +1,7 @@
+try {
+  // Works on same origin.
+  postMessage('done');
+} catch(err) {
+  // Works cross-origin.
+  postMessage('done', '*');
```

Comment:
When on same origin: ```postMessage('done', '*');``` produces this interesting error:

```Uncaught TypeError: Failed to execute 'postMessage' on 'WorkerGlobalScope': The 2nd argument is neither an array, nor does it have indexed properties.```

Bug, no bug? Haven't yet tried on FF.